stable/redis/templates/redis-with-sentinel-svc.yaml

```diff
@@ -8,6 +8,9 @@ metadata:
     chart: {{ template "redis.chart" . }}
     release: {{ .Release.Name }}
     heritage: {{ .Release.Service }}
+    {{- if .Values.master.service.labels }}
```

Comment:
This should be the sentinel labels not the master.

Reply:
Good catch! Fixed.